Vinay/palindrome.py

```diff
@@ -0,0 +1,6 @@
+s = input("Enter the sentence:")
+if s == s.replace("",""):

```

Comment:
what about other special characters?

Reply:
I ran the code with A man, a plan, a canal: Panama$ it is showing me current output. So I think it is working.